content/docs/started/k8s/kfctl-existing-arrikto.md

```diff
@@ -65,7 +199,7 @@ To get the Istio Gateway IP, run:
 kubectl get svc -n istio-system istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
 ```
 
-Get the IP and open it in a browser: `https://<ip>/`.
+Get the IP and open it in a browser: `https://<LoadBalancerIP>/`.
```

Comment:
s/IP/IP address

Reply:
Thanks Sarah, I changed to `<LoadBalancerIP address>`

@yanniszark what do you think?

Reply:
I found two more occurrences and amended accordingly